src/apps/filters/h3.c

```diff
@@ -110,6 +113,38 @@ bool latLngToCellCmd(int argc, char *argv[]) {
     return true;
 }
 
+bool cellToBoundaryCmd(int argc, char *argv[]) {
+    Arg cellToBoundaryArg = {
+        .names = {"cellToBoundary"},
+        .required = true,
+        .helpText =
+            "Convert an H3 cell to an array of latitude/longitude coordinates "
+            "defining its boundary",
+    };
+    Arg helpArg = ARG_HELP;
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&cellToBoundaryArg, &helpArg, &cellArg};
+    if (parseArgs(argc, argv, 3, args, &helpArg,
+                  "Convert an H3 cell to an array of latitude/longitude "
+                  "coordinates defining its boundary")) {
+        return helpArg.found;
+    }
+    CellBoundary cb;
+    int err = H3_EXPORT(cellToBoundary)(cell, &cb);
+    if (err) {
+        return false;
+    }
+    // TODO: Is it better to use brackets or parentheses for the output?
+    printf("[\n");
+    for (int i = 0; i < cb.numVerts; i++) {
+        LatLng *ll = &cb.verts[i];
+        printf("  [%.10lf, %.10lf],\n", H3_EXPORT(radsToDegs)(ll->lat),
+               H3_EXPORT(radsToDegs)(ll->lng));
+    }
+    printf("]\n");
```

Comment:
This output is *not quite* JSON-ish, at the moment. Do we want it to conform to JSON? (If so, we should change the other outputs, too.) Or do we want to focus on human readability? (If so, should this be switched to parentheses between coordinates and no outer parens/brackets?)

Reply:
I would suggest offering WKT and GeoJSON as output formats. WKT is I think close enough to this kind of "human readable" form.

Reply:
Getting back to this again with some spare time. So I am planning on just making it all WKT in this PR and make JSON formatting a follow-up PR. (Or I could reverse that ordering if you prefer.)

I've checked a few WKT definitions, but [they don't specify lat, lng ordering; they just call it X and Y](https://www.ibm.com/docs/en/i/7.4?topic=formats-well-known-text-wkt-format) perhaps to avoid the lat, lng vs lng, lat holy war? But not sure which ordering we want to use here. I'll stick with lat, lng for now and if there's any reason to swap, let me know.

Reply:
I believe it needs to be longitude, latitude, which corresponds to X, Y. I tried visualizing the output on e.g. https://wktmap.com/ and it does not look right if it is not longitude, latitude, so we would not be compatible with existing tooling.